tests/MongoDB.Driver.Core.Tests/Core/Authentication/ScramSha256AuthenticatorTests.cs

```diff
@@ -302,6 +308,54 @@ public void Constructor_should_throw_an_ArgumentNullException_when_credential_is
             subject._cache()._cacheKey().Should().NotBe(null);
             subject._cache()._cachedEntry().Should().NotBe(null);
         }
+
+#if !NETCOREAPP1_1
+        [Theory]
+        [ParameterAttributeData]
+        public void Authenticate_should_work_regardless_of_culture(
+            [Values("da-DK", "en-US")] string cultureName,
+            [Values(false, true)] bool async)
+        {
+            ProtectCulture(() =>
+            {
+                var randomStringGenerator = new ConstantRandomStringGenerator("a");
```

Comment:
as soon as we will know a bigger number of weird cases for string comparison, this line can be updated to be able to use not only `a` (and `aa` in line 327)